verification/curator-service/ui/src/components/NewCaseForm.tsx

```diff
@@ -118,7 +119,6 @@ class NewCaseForm extends React.Component<Props, NewCaseFormState> {
                 },
                 location: {
                     ...values.location,
-                    ...{ query: values.locationQuery },
                 },
```

Comment:
this can simply be `location: values.location` now

Reply:
Done